node/src/components/small_network/tasks.rs

```diff
@@ -97,14 +97,18 @@ where
         .set_nodelay(true)
         .map_err(ConnectionError::TcpNoDelay)?;
 
-    let mut transport = tls::create_tls_connector(context.our_cert.as_x509(), &context.secret_key)
-        .and_then(|connector| connector.configure())
-        .and_then(|mut config| {
-            config.set_verify_hostname(false);
-            config.into_ssl("this-will-not-be-checked.example.com")
-        })
-        .and_then(|ssl| SslStream::new(ssl, stream))
-        .map_err(ConnectionError::TlsInitialization)?;
+    let mut transport = tls::create_tls_connector(
+        context.our_cert.as_x509(),
+        &context.secret_key,
+        context.keylog.clone(),
+    )
+    .and_then(|connector| connector.configure())
+    .and_then(|mut config| {
+        config.set_verify_hostname(false);
+        config.into_ssl("this-will-not-be-checked.example.com")
```

Comment:
Why will this not be checked? Suggest a comment here that describes behavior.

Reply:
This will not be checked due to the fact that we do not have domain names. TLS verification is usually certificate against domain and issuer. There's no domain to be checked at all, our "name" is the certificates fingerprint, which is unforgeable.

This code did not change in this PR as well, it's just a formatting change due to the line above getting an extra parameter.